### PR TITLE
add three options: --ssl-key-file --ssl-cert-file --ssl-ca-file

### PR DESCRIPTION
**Pull request by tcler**
test pass:
'''
[tcler@ws pems]$ stomp -H messaging-devops-broker02.web.prod.***.com -P 61612 --ssl-key-file key.pem --ssl-cert-file cert.pem --ssl-ca-file ca.pem --ssl -V
('Attempting connection to host %s, port %s', 'messaging-devops-broker02.web.prod.***.com', '61612')
Sending {cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}
CONNECTED
session: ID:messaging-devops-broker02.web.prod.***.com-40116-1538113127351-3:29629
version: 1.1
server: ActiveMQ/5.11.0.***-620133
heart-beat: 0,0
>

quit
'''

Signed-off-by: Jianhong Yin <yin-jianhong@163.com>

### CODE REVIEW

File: stomp/__main__.py [hunk 1]
```diff
@@ -20,9 +20,16 @@
                             Listen for messages on a queue/destination
   -V, --verbose             Verbose logging "on" or "off" (if on, full headers
                             from stomp server responses are printed)
-  --ssl                     Enable SSL connection
   --heartbeats=<heartbeats> Heartbeats to request when connecting with protocol >=
                             1.1 (two comma separated integers required) [default: 0,0]
+  --ssl                     Enable SSL connection
+  --ssl-key-file=<cert-file>
```
Comment (jasonrbriggs) [inline stomp/__main__.py]: @tcler This doesn't work with docopt (just tested it and it doesn't parse).  The variable names probably need to be unique.

File: stomp/__main__.py [hunk 2]
```diff
@@ -530,7 +537,7 @@ def main():
     heartbeats = tuple(map(int, arguments['--heartbeats'].split(",")))
 
     st = StompCLI(arguments['--host'], arguments['--port'], arguments['--user'], arguments['--password'], arguments['--protocol'],
-        prompt, arguments['--verbose'], arguments['--ssl'], heartbeats)
+        prompt, arguments['--verbose'], heartbeats, arguments['--ssl'], arguments['--ssl-key-file'], arguments['--ssl-cert-file'], arguments['--ssl-ca-file'])
```
Comment (jasonrbriggs) [inline stomp/__main__.py]: I think you need to reference the variable names, not the argument name (i.e. `arguments['<ssl-key-file>']`)

Comment (tcler) [reply stomp/__main__.py]: thanks jason, will send new patch.

File: stomp/__main__.py (hunk 1 above)
Comment (tcler) [reply stomp/__main__.py]: Oops, sorry will send new patch.

**Pull request merged**